builds/releasenotes.md

```diff
@@ -6,14 +6,29 @@ Known Issues In Current Release
     - Round trip for exporting pointclouds doesn't always give the same model
     - Export to UDS doesn't copy all metadata
   - Map mode is unavailable
-  - Lots of minor UI sizing/positioning issues
   - Displacement compare
     - Calculation doesn't always work correctly
     - Sits at 0% for a long time while during initial processing
-  - Lack of prompts for destructive actions
   - Some LAZ files fail to convert
   - Some SLPK files fail to render correctly and SLPK pointcloud is unsupported
 
+Version 0.6.1 (Minor Bug Fix Release)
+  - Added helper system for translations
+    - Updated Chinese translations
+    - Recieved Japanese translation from Kimoto Group <support-geoverse@kimoto.co.jp>
+  - Fixed lots of issues with the Emscripten build
```

Comment:
Should this say something similar for the Metal build? It no longer crashes on boot/login, it's not good enough to ship yet but we're closer.